smartapps/bangali/rooms-child-app.src/rooms-child-app.groovy

```diff
@@ -1766,10 +1775,24 @@ def	nightButtonPushedEventHandler(evt)     {
     if (nightSwitches && roomState == 'asleep')     {
         unscheduleAll("night button pushed handler")
         def switchValue = nightSwitches.currentValue("switch")
-        if (switchValue.contains('on'))
-            nightSwitchesOff()
-        else
-            dimNightLights()
+        if (nightButtonAction == "1")
+        {
+        	log.debug "action 1"
```

Comment:
please change all log.debug to ifDebug, so when we shut off debug user doesnt see debug messages